packages/react/src/ActionList/ActionList.examples.stories.tsx

```diff
@@ -327,7 +327,7 @@ const SortableItem: React.FC<React.PropsWithChildren<SortableItemProps>> = ({opt
   return (
     <ActionList.Item
       role={role}
-      ref={element => dragRef(element) && dropRef(element)} // merge refs
+      ref={(element: HTMLLIElement) => dragRef(element) && dropRef(element)} // merge refs
```

Comment:
Is this required? Does it throw a TS error without this? (asking to check if we would need to do this in dotcom as well?)

Reply:
Just removed it, as it wasn't required for TS!